Client/Frontend/AuthenticationManager/AppAuthenticator.swift

```diff
@@ -9,39 +9,35 @@ import LocalAuthentication
 
 class AppAuthenticator {
     static func presentAuthenticationUsingInfo(authenticationInfo: AuthenticationKeychainInfo, touchIDReason: String, success: (() -> Void)?, cancel: (() -> Void)?, fallback: (() -> Void)?) {
```

Comment:
Nit: Now that this method is used for touch ID purposes only, I would rename it to `presentTouchAuthenticationUsingInfo` to help differentiate it from the other passcode authentication methods.